streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -281,10 +280,27 @@ private void shutdown() {
         log.info("Stream thread shutdown complete [" + this.getName() + "]");
     }
 
+    /**
+     * Compute the latency based on the previous marked timestamp,
+     * and update the marked timestamp with the current system timestamp.
+     *
+     * @param now Previous marked timestamp
+     * @return latency
+     */
+    private long computeLatency(long now) {
+        long then = now;
+        now = time.milliseconds();
+
+        return Math.max(now - then, 0);
```

Comment:
Why call it `now` and not `then` (or `previous`); `time.milliseconds()` give the current timestmap, ie, "now"